README.md

```diff
@@ -280,18 +280,23 @@ A build Task, e.g. one that produces a container image, must abide by both `all-
 
 ## Task Migration
 
-Task migrations are Bash scripts defined in version-specific task
-directories. In general, a migration consists of a series of `yq` commands that
-modify pipeline in order to work with the new version of task. Developers can
-do more with task migrations on the pipelines, e.g. add/review a task,
-add/remove/update task parameters, change execution order of a task, etc.
+Task migrations allow task maintainers to introduce changes to Konflux standard
+pipelines according to the task updates. By creating migrations, task
+maintainers are able to add/remove mandatory task to/from pipelines,
```

Comment:
I'd put task adding/removal at the end since it's less often case.

Reply:
Updated.